src/Sylius/Bundle/AdminBundle/templates/product_review/form/sections/details/rating.html.twig

```diff
@@ -0,0 +1,7 @@
+{% if form is not defined %}
```

Comment:
Why might the form be undefined?

Reply:
Because the form is passed in the `hookable_metadata.context`